docs/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!--- If applicable, issue number goes here, e.g. GODRIVER-ABCD -->
+
+## Description
+<!--- A brief description of the pull request. -->
+
+## Type of change
```

Comment:
Suggest removing the `Type of change` section. I expect a "Breaking change" would be part of a major release and very rare.